ada-project-docs/wave_04.md

```diff
@@ -0,0 +1,57 @@
+# OPTIONAL Wave 4: Enhancements
+
+These really are **optional** - if you've gotten here and you have time left, that means you're moving speedy fast!
+
+### More Inventory Management
+All these endpoints should support all 3 query parameters. All fields are sortable.
```

Comment:
This might not be necessary, but it could be helpful to specify we mean the query parameters from wave 3.
```suggestion
All these endpoints should support all 3 query parameters from Wave 3. All fields are sortable.
```